src/components/alert.js

```diff
@@ -33,14 +33,7 @@ import globalize from '../scripts/globalize';
             });
 
             options.buttons = items;
-
-            return dialog.show(options).then(function (result) {
-                if (result === 'ok') {
-                    return Promise.resolve();
-                }
-
-                return Promise.reject();
-            });
+            return dialog.show(options);
```

Comment:
Doing a quick search, it looks like there are other places that would be handling this promise rejection. How did you confirm this is not needed?

Reply:
I actually found the opposite, can you point to an example? It looked to me like there were two or three methods with the same name, and this one wasn't using the promise rejection anywhere. It was actually another source of unhandled exceptions whenever the dialog was closed without explicitly clicking the OK button. The only use I saw was the one to supress an unhandled exception.